deliveries/starlay_finance-milestone_1.md

```diff
@@ -0,0 +1,16 @@
+# Milestone Delivery :mailbox:
+
+**The invoice has been send out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/smart-contract-bounty/Support-Docs/blob/master/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [Starlay Finance](https://github.com/Chidasan/Ecosystem-Grants/blob/master/applications/starlay_finance.md).
+
+**Deliverables**
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0a. | License | [License](https://github.com/starlay-finance/starlay-protocol-wasm/blob/f8465b9e79b7566610663fd5f2971a9c0eeac8f9/LICENSE.md) | 　|
+| 0b. | Documentation | [Documentation](https://docs.starlay.finance) |　 |
+| 1. | Deposit function | [Deposit function](https://github.com/starlay-finance/starlay-protocol-wasm/blob/f8465b9e79b7566610663fd5f2971a9c0eeac8f9/logics/impls/pool/mod.rs#L337)  | |
```

Comment:
For future reviewers, here are some of the related TS E2E tests:

- Borrow
    - https://github.com/starlay-finance/starlay-protocol-wasm/blob/f8465b9e79b7566610663fd5f2971a9c0eeac8f9/tests/Pool.spec.ts#L381
- Mint
    - https://github.com/starlay-finance/starlay-protocol-wasm/blob/f8465b9e79b7566610663fd5f2971a9c0eeac8f9/tests/Pool.spec.ts#L111
- Redeem
    - https://github.com/starlay-finance/starlay-protocol-wasm/blob/f8465b9e79b7566610663fd5f2971a9c0eeac8f9/tests/Pool.spec.ts#L206
- Repay Borrow
    - https://github.com/starlay-finance/starlay-protocol-wasm/blob/f8465b9e79b7566610663fd5f2971a9c0eeac8f9/tests/Pool.spec.ts#L472